atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -234,11 +238,15 @@ private void runOnceForTable(TableToSweep tableToSweep) {
                     batchConfig,
                     startRow);
             long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
-            log.debug("Swept {} unique cells from {} starting at {}"
+            log.info("Swept {} unique cells from {} starting at {}"
                             + " and performed {} deletions in {} ms"
                             + " up to timestamp {}.",
-                    results.getCellTsPairsExamined(), tableRef, startRowToHex(startRow),
-                    results.getStaleValuesDeleted(), elapsedMillis, results.getSweptTimestamp());
+                    SafeArg.of("", results.getCellTsPairsExamined()),
+                    UnsafeArg.of("table name", tableRef),
+                    SafeArg.of("row hex", startRowToHex(startRow)),
```

Comment:
IIRC, this is not necessarily a row key, but the old way paging worked would cause this value to be the next byte array value that follows the value of the last row from a previous page. Hadn't looked into the new sweep code, but probably best to err on the side of caution and make it unsafe.